app/controllers/application_controller.rb

```diff
@@ -37,6 +38,10 @@ def redirect_user_with_no_organisation
 
 protected
 
+  def show_navigation_bars
```

Comment:
I like this better, but there is this bit of logic in there that is very local to one controller. I think that, if you didn't know this was here, figuring out why the layout changes for this particular action would be really tricky.

I was thinking you could change this method back to:
```
def show_navigation_bars
  user_signed_in?
end
```
and then in the invitations controller you can overload this method:
```
  def show_navigation_bars
    return false if action_name=="invite_second_action"
    super
  end
```
That way, when you're looking at the `invite_second_admin` action, you can easily see why the navigation bars don't show.

What do you think?